Makefile

```diff
@@ -688,7 +686,7 @@ test-addons-clean:
 	$(MAKE) test-node-api-clean
 
 .PHONY: test-with-async-hooks
-test-with-async-hooks:
+test-with-async-hooks: all ## Run tests with async hooks.
```

Comment:
The addition of `all` dependency looks unrelated

Reply:
Oops!